README.md

```diff
@@ -1 +1,40 @@
-# mobile-version-skeleton
\ No newline at end of file
+# Setup and mobile version skeleton
+
+>Portfolio setup and mobile version skeleton front-end page.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+## Getting Started
+
+git clone https://github.com/Ghilain/mobile-version-skeleton.git
+cd mobile-version-skeleton
+
+## Authors
+
+👤 **Ghilain Ishimwe**
+
+- GitHub: [@githubhandle](https://github.com/Ghilain)
+- Twitter: [@twitterhandle](https://twitter.com/GhilainIshimwe)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ghilain-ishimwe)
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] OPTIONAL: This link is broken, please consider fixing it by adding `.MIT` file.